### DISCUSSION TITLE
Why the nuxt loading indicator not is removed?

### DISCUSSION THREAD

**Discussion by mitraecp** [nuxt/nuxt#26536] [Questions]
I have a scenario where the loading incidator does not disappear

is just a page **/marketplace**

a menu with go to **/marketplace/templates/crm-and-marketing** or  **/marketplace/templates/finantial**

then, when useer is inside this routes, display a button to back to  **/marketplace**
and when i do this, all works OK, but the loading display and never leaves

**Comment by TheAlexLichter** [comment]
Can you provide a reproduction? Also make sure your Nuxt version is up to date, a similar bug was fixed in a previous version
